hera_cal/nucal.py

```diff
@@ -1348,9 +1379,9 @@ def _estimate_degeneracies(self, data, model, wgts):
 
         return amplitude, tip_tilt
     
-    def post_redcal_nucal(self, data, data_wgts, cal_flags={}, spatial_estimate_only=True, linear_solver="lu_solve", linear_tol=1e-12, share_fg_model=False, 
+    def post_redcal_nucal(self, data, data_wgts, cal_flags={}, spatial_estimate_only=False, linear_solver="lu_solve", linear_tol=1e-12, share_fg_model=False, 
```

Comment:
Linear_tol vs. alpha, and also default value, and also documentation

Reply:
Fixed